ios/Plugin/HttpRequestHandler.swift

```diff
@@ -190,10 +190,9 @@ class HttpRequestHandler {
         }
 
         let urlRequest = request.getUrlRequest()
-        
         let session = URLSession(configuration: URLSessionConfiguration.default, delegate: allowAllDelegate, delegateQueue: nil)
-        
         let task = session.dataTask(with: urlRequest) { (data, response, error) in
+            session.invalidateAndCancel()
```

Comment:
Why is this needed? And if it is needed, wouldn't `finishTasksAndInvalidate()` be the right thing to do?

Reply:
Change happened here (without any reasoning): https://github.com/capacitor-community/http/pull/157/commits/eaaedecd8921c358447fa66314b36b1151da586c

Reply:
Yes, I also saw this and merged it (this seams to be an improvement in the HTTP plugin). I couldn't figure out, why this is needed, but i removed the semicolon at the end.